pants.toml

```diff
@@ -97,6 +98,11 @@ inits = true
 [python-protobuf]
 mypy_plugin = true
 
+[autoflake]
```

Comment:
How do we actually invoke this? Is it something that should be added to `make format-python`?